lib/data/argot/traject_config.rb

```diff
@@ -60,7 +60,8 @@
 
 unless settings["override"].include?("isbn")
   to_field "isbn" do |rec, acc|
-    Traject::MarcExtractor.cached(settings["specs"][:isbn], :alternate_script => false).each_matching_line(rec) do |field, spec, extractor|
+    Traject::MarcExtractor.cached(settings["specs"][:isbn], :alternate_script => false)
```

Comment:
It's fine to break up this line but I'd prefer it if the method calls were aligned:

```
    Traject::MarcExtractor.cached(settings["specs"][:isbn], :alternate_script => false)
                          .each_matching_line(rec) do |field, spec, extractor|
```